staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go

```diff
@@ -114,6 +114,11 @@ func transformResponseObject(ctx request.Context, scope RequestScope, req *http.
 				return
 			}
 
+			if scope.TableConvertor == nil {
+				err = newNotAcceptableError("scope.TableConvertor is nil")
```

Comment:
will this end up in the response? This should be an error understandable by the user, or http 500 if it's really internal.